tests/test_const_table.py

```diff
@@ -25,6 +25,97 @@ def test_const_table(db: gp.Database):
 def test_table_getitem(db: gp.Database):
     rows = [(1,), (2,), (3,)]
     t = gp.values(rows, db=db)
-    t = t.save_as("const_table", column_names=["id"])
+    t = t.save_as("const_table", temp=True, column_names=["id"])
     c = t["id"]
     assert str(c) == (t.name + ".id")
+
+
```

Comment:
`join` is a big and complex feature which worth a separate test file. I propose to create a new file `test_join.py` and organize the test like the below:

**Tests on `_join` method.**
Since it is the base method for all the "join" APIs, we could use those tests to verify common things like the query result columns based on different input combinations. In those tests, the output tuples are not important, and tuples verification should be coved by other tests.

**Tested on individual public "join" method.**
For those tests, the result tuples actually matter. And since different joins may have different characteristics, the tests should cover all the differences of those.


"join" could be complex, to prevent this from becoming a huge PR, choosing a few public methods to implement as a start might be a good idea. But that totally depends on you.